src/InstagramClone.Api/Database/InstagramCloneDbContext.cs

```diff
@@ -0,0 +1,37 @@
+﻿using InstagramClone.Api.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace InstagramClone.Api.Database
+{
+    public class InstagramCloneDbContext : DbContext
+    {
+        public InstagramCloneDbContext(): base()
+        {
+
+        }
+
+       
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<User>().HasKey(u => u.Id);
+            modelBuilder.Entity<Post>().HasKey(p => p.Id);
+            modelBuilder.Entity<PostReaction>().HasKey(pr => pr.Id);
+            modelBuilder.Entity<UserFollower>().HasKey(userFollower => userFollower.FollowedUserId);
+
```

Comment:
create a composite key using the 2 foreign keys on this `UserFollower` table. 

See https://www.learnentityframeworkcore.com/configuration/fluent-api/haskey-method on how to create a composite key

Reply:
Fixed